go/worker/keymanager/watcher.go

```diff
@@ -284,6 +284,32 @@ func (w *rtNodeWatcher) watch(ctx context.Context) {
 	}
 	defer sub.Close()
 
+	// Maintain a list of allowable nodes in addition to the committee members.
+	observerNodes := map[signature.PublicKey]struct{}{}
+
+	nodeCh, nodeSub, err := w.consensus.Registry().WatchNodes(ctx)
```

Comment:
I guess we could remove `watcher.WatchNodeUpdates()` as now we have to watch all nodes anyway. But I'm not sure :thinking:

Reply:
Yeah, one would just need to introduce a separate `computeNodes` map that is updated during epoch transitions and then refresh it below in `nodeCh` handler.

Because that is basically what the "watcher" is doing anyway and I think that would actually be cleaner.